doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+| `edgehub_gettwin_duration_seconds_sum`                      | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `source` (Operation source)<br> `id` (Module ID)                                                                | Sum of time taken for get twin operations                                                                                                                                                                | summary |
+| `edgehub_gettwin_duration_seconds_count`                    | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `source` (Operation source)<br> `id` (Module ID)                                                                | Measurement count                                                                                                                                                                                        | summary |
+| `edgehub_gettwin_duration_seconds`                          | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `source` (Operation source)<br> `id` (Module ID)<br> `quantile`(Percentile [50, 90, 95, 99, 99.9, 99.99])       | P50, P90, P95, P99, P99.9 and P99.99  time taken for get twin operations.  Values may be reported as `NaN` if no  new measurements are reported for a certain  period of time (currently 10 minutes)     | summary |
+| `edgehub_message_send_duration_seconds_sum`                 | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `from` (Message source)<br> `to` (Message destination)                                                          | Sum of time taken to send a messsage                                                                                                                                                                     | summary |
+| `edgehub_message_send_duration_seconds_count`               | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `from` (Message source)<br> `to` (Message destination)                                                          | Measurement count                                                                                                                                                                                        | summary |
+| `edgehub_message_send_duration_seconds`                     | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `from` (Message source)<br> `to` (Message destination)<br> `quantile`(Percentile [50, 90, 95, 99, 99.9, 99.99]) | P50, P90, P95, P99, P99.9 and P99.99 time taken to send a message. Values may be reported as `NaN`  if no new measurements are reported for a  certain period of time (currently 10 minutes)             | summary |
```

Comment:
I think for summary we should provide the uber metric, and then state that this will emit the following metrics - sum, count, and percentiles (instead of each separately). 
For example edgehub_message_send_duration_seconds is the summary metric (edgehub_message_send_duration_seconds_count itself is not very useful).